flink/v1.20/flink/src/test/java/org/apache/iceberg/flink/TestIcebergConnector.java

```diff
@@ -256,43 +256,6 @@ public void testCatalogDatabaseConflictWithFlinkDatabase() {
         .hasMessageStartingWith("Could not execute CreateTable in path");
   }
 
-  @TestTemplate
-  public void testConnectorTableInIcebergCatalog() {
-    // Create the catalog properties
-    Map<String, String> catalogProps = Maps.newHashMap();
```

Comment:
Why is this test removed?

Reply:
This is testing the check mentioned in https://github.com/apache/iceberg/pull/12116#discussion_r1933510999

Fail creating a table in Iceberg Catalog if connector=iceberg is specified in the Option. As the check is been deleted, i removed this test case.

Reply:
I think that this is still a valid check in most cases. Only not valid when the table is created with "CREATE TABLE.. LIKE" and only if the source table is an iceberg table.
Do I miss something?